tests/acceptance/current-url-test.js

```diff
@@ -22,4 +22,15 @@ test('visiting /release-url', function (assert) {
     let currentVersion = page.get('currentVersion');
     assert.equal(find('.ember-basic-dropdown-trigger').text().trim(), currentVersion);
   });
-});
\ No newline at end of file
+});
+
+test('visiting / redirects you to /release', function (assert) {
+  visit('/');
+  let page = this.application.__container__.lookup('service:page');
+  andThen(function () {
+    assert.equal(currentURL(), "/release");
+
+    let currentVersion = page.get('currentVersion');
+    assert.equal(find('.ember-basic-dropdown-trigger').text().trim(), currentVersion);
```

Comment:
Do we need to check canonical urls on meta tags as well?

Reply:
we haven't implemented canonical URLs yet on the Guides app... is this even in the current infrastructure? 🤔

Reply:
oh this is interesting... it looks like it *used* to be part of the infrastructure. Older snapshots have canonical links but newer ones dont... very interesting

This is definatly something that is on the todo list but it was agreed to not be a blocker for this release 👍

Reply:
In the existing app, `/release` doesn't exist, so canonical urls don't exist either.  But since we're introducing this its probably good to introduce canonical urls too. If not we might temporarily screw up search engine results a bit.

Reply:
I am happy to discuss considering that as a blocker for the release but I'm not sure it is a blocker for **this** PR 🤔

Reply:
May be open a related issue for the implementation and then merge this PR?

Reply:
https://github.com/ember-learn/guides-app/issues/55 🎉 💪 #letsdothis